src/tests/examples/example_test.py

```diff
@@ -51,6 +53,8 @@ def verify_solution(study_path, expected_values, expected_investment_solution):
 
     solution = json_data["solution"]
     investment_solution = solution["values"]
+    
+    json_file.close()
```

Comment:
we could use 
```python
with open(str(json_path), 'r') as json_file:
    json_data = json.load(json_file)
```
wthout the need to `close` the file

Reply:
OK but what will happen if there is an assert raised ?

I added the close statement because in Windows we couldn't launch another test because json output file was already used.